sdjournal/journal.go

```diff
@@ -651,6 +652,43 @@ func (j *Journal) GetDataValue(field string) (string, error) {
 	return strings.SplitN(val, "=", 2)[1], nil
 }
 
+// GetDataBytes gets the data object associated with a specific field from the
+// current journal entry.
+func (j *Journal) GetDataBytes(field string) ([]byte, error) {
+	sd_journal_get_data, err := j.getFunction("sd_journal_get_data")
```

Comment:
I'm tempted to say this is an overoptimisation (although I do think that the interface should probably have been `[]byte` in the first place). In any case, if we do add this, the code is 99% identical between the functions - can you factor it out so they can share it?